flashinfer/sampling.py

```
@@ -333,10 +367,30 @@ def min_p_sampling_from_probs(
             offset_val,
         )
+        if return_valid:
+            return samples, valid
         return samples
 
-    # torch library for top_k_top_p_sampling_from_probs
+    @register_fake_op("flashinfer::min_p_sampling_from_probs")
+    def _fake_min_p_sampling_from_probs(
+        probs: torch.Tensor,
+        indices: Optional[torch.Tensor],
+        maybe_min_p_arr: Optional[torch.Tensor],
```

Ruff (0.15.1)

[warning] Unused function argument: `maybe_min_p_arr`

(ARG001)